pages/blocks/headerwithlockup/headerwithlockup.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const header = document.querySelector('.headerwithlockup div');
```

Comment:
if you `default export` a function from this block js, it will be invoked with the `.headerwithlockup` div as the first argument, then you don't need to query document

not technically required, since the rest of the file runs as a script, but that style would follow the other blocks (and the other helix sites)